lib/WorkflowSraDownload.groovy

```diff
@@ -0,0 +1,19 @@
+/*
+ * This file holds functions specific to the workflow/sra_download.nf in the nf-core/rnaseq pipeline
+ */
+
+class WorkflowSraDownload {
+
+    /*
+     * Print a warning after SRA download has completed
+     */
+    public static void sraDownloadWarn(log) {
+        log.warn "=============================================================================\n" +
```

Comment:
Might be worth mentioning here the strandedness, I know is nicely explained in the `usage.md` ("public databases don't reliably hold information such as strandedness information so you may need to amend these entries too") so up to you...

Reply:
I did have that info in originally but I removed it to make it more generic so it will be easier to copy and paste this across when we add an independent workflow for this. For example, the strandedness stuff isn't applicable to viralrecon where we are using exactly the same workflow. Added back in anyway https://github.com/nf-core/rnaseq/pull/622/commits/41dbf0ec3fd96550c728de94cc7bcf6d67d9c3c9